<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomCurvePage</class>
 <widget class="QWidget" name="CustomCurvePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton,QPushButton{
font: 10pt &quot;微软雅黑&quot;;
color: rgb(97, 97, 97);
background-color: rgb(162, 162, 162);
border-style:none;
}
QToolButton:hover,QPushButton:hover{
font: 14pt &quot;微软雅黑&quot;;
color: rgb(97, 97, 97);
background-color: rgb(214, 214, 214);
border-style:none;
}

QToolButton:checked,QPushButton:pressed{
font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(97, 97, 97);
background-color: rgb(227, 227, 227);
border-style:none;
}
QWidget#widget{
background-color: rgb(94, 94, 94);
}
/*QWidget#DialogPickCurve{
background-color:rgba(0,0,0,0);
}
QWidget#page_user{
background-color:rgba(0,0,0,0);
}*/

QTreeView,QTableView{
	show-decoration-selected: 1;
	border:none; 
  }
QListView {
	show-decoration-selected: 1;
	border:none; 
	
	background-color: rgb(238, 238, 238);
  }
QTreeView::item,QTableView::item,QListView::item {
      border: 1px solid #d9d9d9;
      border-left-color: transparent;
      border-top-color: transparent;
  }
QTreeView::item:hover,QTableView::item:hover,QListView::item:hover {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e7effd, stop: 1 #cbdaf1);
      border: 1px solid #bfcde4;
  }
QTreeView::item:selected,QTableView::item:selected,QListView::item:selected {
      border: 1px solid #567dbc;
  }
QTreeView::item:selected:active,QTableView::item:selected:active,QListView::item:selected:active{
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6ea1f1, stop: 1 #567dbc);
  }
QTreeView::item:selected:!active,QTableView::item:selected:!active,QListView::item:selected:!active {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6b9be8, stop: 1 #577fbf);
  }

QComboBox{
	border-radius: 3px;
	padding: 1px 10px 1px 5px;
	border: none;
	background-color:transparent;
}
QComboBox:hover{
	border-radius: 3px;
	padding: 1px 10px 1px 5px;
	border: 1px solid #636363;
	background-color: rgb(170, 255, 255);
}
QComboBox::drop-down{
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 15px; 
	border-left-width: 1px;
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
	border-left-color: transparent;
}

QComboBox::down-arrow{
 
	image: url(:/array_down.png);
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>2</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>CurveName:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_curveName">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Note:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_note">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="treeWidget_input">
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Unit</string>
          </property>
          <property name="text">
           <string>factor coefficient</string>
          </property>
          <item>
           <property name="text">
            <string>undefine</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Variable Input</string>
          </property>
          <property name="text">
           <string>nickname</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Const Input</string>
          </property>
          <property name="text">
           <string>nickname</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="btn_add">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_remove">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>example:sin(_pi*V0/360)+V1*0.05+C0</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QsciScintilla" name="textEdit"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_new">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>New</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_save">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_status">
         <property name="text">
          <string>add curve </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn_hide">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>120</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="UTreeWidget" name="treeWidget_variable">
         <column>
          <property name="text">
           <string>name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>value</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>type</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="UTreeWidget" name="treeWidget_const">
         <column>
          <property name="text">
           <string>name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>value</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>type</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">utreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QTextEdit</extends>
   <header location="global">qsciscintilla.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
